platform/accessibility/guidance/staging-review-processes.md

```diff
@@ -13,16 +13,16 @@ Staging reviews test the [happy path](https://en.wikipedia.org/wiki/Happy_path)
 - [ ] Color tests for proper contrast and colorblindness - [read more about contrast and colorblindness](#color-tests)
 - [ ] Content zoomed to 200%, 300%, 400% - [read more about zoomed layouts](#content-resize-check)
 - [ ] Keyboard navigation - [read more about navigating with the keyboard](#keyboard-navigation-check)
-- [ ] Screen reader tests (VoiceOver + Safari or NVDA + Firefox) - [read more about getting started with screen readers](#screen-reader-tests)
 
 ## Recommended accessibility tests
```

Comment:
Rachael mentioned calling these "Advanced accessibility tests" and the previous section "Baseline" - Just putting that there for you to consider, not a required change I don't think.

Reply:
I like that. I fudged a little bit and used the term "foundational", but I've been baking that into conversations so feel it works here too.

Reply:
👍